src/docfx.website.themes/default/ManagedReference.common.js

```diff
@@ -52,7 +54,13 @@ function groupChildren(model, typeChildrenItems) {
   var grouped = {};
 
   model.children.forEach(function (c) {
-    var type = c.type.toLowerCase();
+    if (c.isEii) {
+      var type = "eii";
+    } else if (c.isExtensionMethod) {
+      var type = "extensionMethod"
```

Comment:
this is extension methods defined in current class. MSDN doesn't show this as a separate category. Do we really want to do this?

---
In reply to: [65029735](https://github.com/dotnet/docfx/pull/338#discussion_r65029735) [](ancestors = 65029735)